eng/pipelines/templates/stages/archetype-ios-release.yml

```diff
@@ -51,13 +51,18 @@ stages:
               - template: ../variables/globals.yml
```

Comment:
FYI @tjprescott based on the release changes I've opened https://github.com/Azure/azure-sdk/issues/2894 and assigned to you to also update the guidelines and automation based on all these changes.